index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css">
+    <title>Document</title>
+</head>
+<body>
+    <main class="container">
+        <div class=info-bar>
+            <img src="img/Dark.png" alt="info">
+        </div>
+        <div class="toolbar toolbar-container">
```

Comment:
- [ ] Please wrap the contents in this toolbar section with a `header` tag instead of a `div`